rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/service_introspection.h

```diff
@@ -23,7 +23,7 @@
 
 #include "rosidl_typesupport_introspection_c/message_introspection.h"
 
-typedef struct ROSIDL_PUBLIC rosidl_typesupport_introspection_c__ServiceMembers
+typedef struct rosidl_typesupport_introspection_c__ServiceMembers
```

Comment:
It's to avoid a warning:

```
In file included from /home/jackie/code/ros2_ws/build/rcl_interfaces/rosidl_typesupport_introspection_c/rcl_interfaces/srv/set_parameters__type_support.c:13:0:
/home/jackie/code/ros2_ws/install/include/rosidl_typesupport_introspection_c/service_introspection.h:32:1: warning: ‘visibility’ attribute ignored on types [-Wattributes]
 } rosidl_typesupport_introspection_c__ServiceMembers;
```

The build worked on Windows without the `ROSIDL_PUBLIC`.